multimedia/tsduck/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           makefile 1.0
+PortGroup           github 1.0
+
+github.setup        tsduck tsduck 3.22-1953 v
+
+categories          multimedia
+license             BSD
+maintainers         {mit.edu:quentin @quentinmit} \
+                    openmaintainer
+description         MPEG Transport Stream Toolkit
+long_description    MPEG Transport Stream Toolkit
+
+homepage            https://tsduck.io/
+platforms           darwin
+
+checksums           sha1    cedc70686415a9062bee8c3b1f43af8ea74f2127 \
+                    rmd160  5a7c0b73ab1b6aa9d88b9d6ecde9befb00747f0f \
+                    sha256  ca4e0af170616eb7ea25b3555a2519b94b8443b3135a4681993b5ae54316f5d9 \
+                    size    13567538
+
+depends_build-append bin:gsed:gsed \
+		     bin:dos2unix:dos2unix
+
+depends_lib-append  port:curl
+
+# PCSC requires pcsc-lite
+# SRT requires libsrt
+# Neither of these are currently available in macports
+set args	    "NOPCSC=true NOSRT=true CFLAGS_EXTRA='-Wno-unknown-warning-option -Wno-unknown-pragmas' CXXFLAGS_EXTRA='-Wno-unknown-warning-option -Wno-unknown-pragmas' LDFLAGS_EXTRA=-Wl,-rpath,@executable_path/../lib"
```

Comment:
Why do you want to set `-rpath`? We usually don't.

Reply:
Makefile.tsduck has
```Makefile
ifdef MACOS
    CFLAGS_INCLUDES += -I$(LIBTSDUCKDIR)/mac
    LDFLAGS_EXTRA += -Wl,-rpath,@executable_path
    SOFLAGS = -install_name '@rpath/$(notdir $@)'
```
which means the executables will only look for their libraries in the same directory as the executable. So, I need to put `$prefix/lib` into `rpath`.

Reply:
Or maybe we should patch Makefile.tsduck so that it doesn't use rpath.